ngraph/core/include/openvino/op/divide.hpp

```diff
@@ -49,6 +49,10 @@ class OPENVINO_API Divide : public util::BinaryElementwiseArithmetic {
     bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
     OPENVINO_SUPPRESS_DEPRECATED_END
     bool has_evaluate() const override;
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate_lower(const HostTensorVector& outputs) const override;
+    bool evaluate_upper(const HostTensorVector& outputs) const override;
+    OPENVINO_SUPPRESS_DEPRECATED_END
```

Comment:
Can we add evaluate methods with runtime tensors instead?

Cc @ilyachur

Reply:
Inside these functions I use evaluate methods of several other operations with HostTensorVector. If I will use ov::runtime::TensorVector, I will need convert it to HostTensorVector and back similar to how it is done in node.cpp. 

@ilya-lavrenov @ilyachur Does it make sense in this case?

Reply:
I call inside these methods several other evaluate methods for different operations with HostTensorVector. So, I will need to convert ov::runtime::TensorVector to HostTensorVector and back as it was done in node.cpp.

@ilya-lavrenov @ilyachur Does it make sense to change these functions now?

Reply:
In general it already should work from the box

Reply:
Evaluate methods with RuntimeTensors requires will automatically workable due to conversion in node.cpp, so I will not re-write code to use TensorVector. Almost all other operations have evaluate methods with HostTensorVector and infrastructure functions like NGRAPH_CHECK, creating tensor from constant works with HostTensor.